src/fprime/cookiecutter_templates/cookiecutter-fprime-deployment/{{cookiecutter.deployment_name}}/Top/{{cookiecutter.deployment_name}}Topology.cpp

```diff
@@ -147,8 +147,10 @@ void setupTopology(const TopologyState& state) {
     regCommands();
     // Project-specific component configuration. Function provided above. May be inlined, if desired.
     configureTopology();
+    // Component-specific configurations using fpp phases. Function provided by autocoder.
+    configComponents(state);
     // Autocoded parameter loading. Function provided by autocoder.
-    // loadParameters();
+    loadParameters();
```

Comment:
git detected a conflict here since we fixed it in another PR - would you be able to resolve it?

Reply:
Done